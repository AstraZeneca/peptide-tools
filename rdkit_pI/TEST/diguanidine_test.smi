NC(=N)NCCNC(=N)N