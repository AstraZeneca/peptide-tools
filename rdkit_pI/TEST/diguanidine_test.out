[16:03:14] WARNING: no name column found on line 0
 
======================================================================================================================================================
pI
---------------------------------
     pI mean  13.31       
         err  0.0         
         std  0.0         
 IPC_peptide  13.31       
     ProMoST  13.31       
 Gauci_calib  13.31       
  Bjellqvist  13.31       
     Rodwell  13.31       
    Grimsley  13.31       
   Thurlkill  13.31       
     Solomon  13.31       
   Lehninger  13.31       
      EMBOSS  13.31       
 
 
======================================================================================================================================================
Q at pH7.4
---------------------------------
Q at pH7.4 mean  2.0         
         err  0.0         
         std  0.0         
 IPC_peptide  2.0         
     ProMoST  2.0         
 Gauci_calib  2.0         
  Bjellqvist  2.0         
     Rodwell  2.0         
    Grimsley  2.0         
   Thurlkill  2.0         
     Solomon  2.0         
   Lehninger  2.0         
      EMBOSS  2.0         
 
 
pH interval with charge between -0.2 and  0.2 for pKa set: IPC_peptide and prediction tool: modified Dimorphite-DL
13.0 - 14.9
 
List of calculated BASE pKa's with the corresponding fragments
smiles or AA, base pka : N=C(N)NCCNC(=N)N 12.0
smiles or AA, base pka : N=C(N)NCCNC(=N)N 12.0
 
List of calculated ACID pKa's with the corresponding fragments
 
List of constantly ionized fragments with the corresponding fragments
